src/common/substrait/src/context.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use substrait_proto::protobuf::extensions::simple_extension_declaration::{
+    ExtensionFunction, MappingType,
+};
+use substrait_proto::protobuf::extensions::SimpleExtensionDeclaration;
+
+#[derive(Default)]
+pub struct ConvertorContext {
+    scalar_fn_names: HashMap<String, u32>,
+    scalar_fn_map: HashMap<u32, String>,
```

Comment:
Maybe this is much clear
```suggestion
    fn_name_to_anchor: HashMap<String, u32>,
    anchor_to_fn_name: HashMap<u32, String>,
```

Reply:
`scalar_fn_names` is a supplement, only `scalar_fn_map` will be serialized. They are not  same